src/services/DealService.ts

```diff
@@ -108,7 +110,52 @@ export class DealService {
     /**
      * deals will take care of themselves on account changes
      */
-    this.getDeals();
+    if (!this.deals) {
```

Comment:
initialize is only called on app startup (in main).  `!this.deals` should always be true.  can we delete this or comment on why it is necessary?